frontend/packages/knative-plugin/src/components/routes/RouteDetailsPage.tsx

```diff
@@ -1,21 +1,74 @@
 import * as React from 'react';
+import { ClipboardCopy } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
 import { useActivePerspective } from '@console/dynamic-plugin-sdk';
-import { DetailsForKind } from '@console/internal/components/default-resource';
+import { Conditions } from '@console/internal/components/conditions';
 import { DetailsPage } from '@console/internal/components/factory';
-import { navFactory } from '@console/internal/components/utils';
+import { RoutesDetailsProps } from '@console/internal/components/routes';
+import {
+  DetailsItem,
+  detailsPage,
+  ExternalLink,
+  navFactory,
+  ResourceSummary,
+  SectionHeading,
+} from '@console/internal/components/utils';
 import { K8sKind, K8sResourceKind, referenceForModel } from '@console/internal/module/k8s';
 import {
   ActionMenu,
   ActionMenuVariant,
   ActionServiceProvider,
   useTabbedTableBreadcrumbsFor,
 } from '@console/shared';
+import { PRIVATE_KNATIVE_SERVING_LABEL } from '../../const';
 import { serverlessTab } from '../../utils/serverless-tab-utils';
 
+const RouteDetails: React.FC<RoutesDetailsProps> = ({ obj: route }) => {
+  const { t } = useTranslation();
+  const isPrivateKSVC =
+    route?.metadata?.labels?.[PRIVATE_KNATIVE_SERVING_LABEL] === 'cluster-local';
+  return (
+    <>
+      <div className="co-m-pane__body">
+        <SectionHeading text={t('knative-plugin~Route details')} />
+        <div className="row">
+          <div className="col-sm-6">
+            <ResourceSummary resource={route} />
+          </div>
+          <div className="col-sm-6">
+            <DetailsItem
+              label={t('knative-plugin~Location')}
+              obj={route}
+              path="status.url"
+              hideEmpty
+            >
+              {isPrivateKSVC ? (
+                <ClipboardCopy isReadOnly hoverTip="Copy" clickTip="Copied">
+                  {route?.status?.url}
+                </ClipboardCopy>
+              ) : (
+                <ExternalLink
+                  href={route?.status?.url}
+                  additionalClassName="co-external-link--block"
+                  text={route?.status?.url}
+                />
+              )}
+            </DetailsItem>
+          </div>
```

Comment:
Just a tiny change please: The `DetailsItem` component renders a dt+dd, which is part of a definition list. But you need to wrap this in a dl yourself.

The ResourceSummary above does this automatically. So its just needed here on the right hand side:

```suggestion
          <div className="col-sm-6">
            <dl>
            <DetailsItem
              label={t('knative-plugin~Location')}
              obj={route}
              path="status.url"
              hideEmpty
            >
              {isPrivateKSVC ? (
                <ClipboardCopy isReadOnly hoverTip="Copy" clickTip="Copied">
                  {route?.status?.url}
                </ClipboardCopy>
              ) : (
                <ExternalLink
                  href={route?.status?.url}
                  additionalClassName="co-external-link--block"
                  text={route?.status?.url}
                />
              )}
            </DetailsItem>
            </dl>
          </div>
```

Reply:
Left side:

![image](https://github.com/openshift/console/assets/139310/02ba335f-7c41-444a-8dd3-690c337ddec4)

Right side:

![image](https://github.com/openshift/console/assets/139310/f238dd7f-3ef1-4811-976a-8a5f6709f5d0)